docs/global/modules/samples/pages/monitoring.adoc

```diff
@@ -0,0 +1,50 @@
+= Monitoring the `model-server`
+:navtitle: Monitoring the `model-server`
+
+Besides the afore mentioned use cases, the samples also include a `model-server` monitoring setup.
+This setup uses the `/metrics` endpoint  xref:core:howto/metrics.adoc[provided by the `model-server` by default].
+
+== Instructions
+
+=== Start the components
+
+.Command to start the monitoring profile
+[source,shell]
+----
+docker-compose --profile monitoring up
+----
+
+This command starts the following components:
+
+* `model-server` at http://localhost:28101/[],
+* Prometheus database at http://localhost:9090/[], and
+* Grafana dashboard at http://localhost:3001/[].
+
+
+NOTE: Make sure you picked the correct instance at the top of the Grafana dashboard.
+
+[NOTE]
+====
+Depending on your operating system or docker setup, you might need to adjust the targets defined in the `prometheus.yml` and `datasource.yml`.
+
+Also, the docker network setup might need adjustments in the `docker-compose.yaml`.
```

Comment:
Which adjustments and why? Apart from conflicting port-forwards with other services on the host, I don't see what could need changes?